pipeline/mutate/mutator_hydrator.go

```diff
@@ -174,6 +220,15 @@ func (a *MutatorHydrator) Mutate(r *http.Request, session *authn.AuthenticationS
 	}
 	*session = sessionFromUpstream
 
+	if cfg.Cache.Ttl != "" {
```

Comment:
Let's make this a private field `ttl time.Duration` and parse it in `func (a *MutatorHydrator) Config(config json.RawMessage) (*MutatorHydratorConfig, error)` :)